src/main/java/uk/co/caprica/picam/Camera.java

```diff
@@ -126,6 +126,7 @@ public boolean open() {
      * @return picture capture handler result
      * @throws CaptureFailedException if an error occurs
      */
+    @SuppressWarnings("UnusedReturnValue")
```

Comment:
Why? The return value has a purpose. Presumably your application doesn't use the return value, but that doesn't mean it should be suppressed for everyone does it?